### DISCUSSION TITLE
css Background image (globaleaks multitenant)

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3954] [Q&A] [answered]
Buongiorno,
sto cercando di impostare un'immagine di background per un sito ma non riesco.
Se passo dal css l'immagine mi viene bloccata per ragioni di sicurezza:
- sia da una fonte esterna (es. dal sito del cliente, ma ne comprendo il motivo)
- sia se uploadato all'interno del sito stesso globaleaks
- sia se uploadato all'interno del sito padre (globaleaks)

Ho provato anche ad usare markdown per impostare l'immagine nella presentation e modificarne il posizionamente via css.
È un sistema un po' estremo e formalmente scorretto ma è l'unico modo che ho trovato.
Questo sistema funziona sul sito padre ma non mi funziona sui siti figli.

C'è un modo per ottenere quanto desiderato?
Grazie

**Comment by maelaborazioni** [comment]
Anche con javascript intervenendo sul DOM la situazione non cambia, mi funziona sul sito padre ma sul sito figlio. 
L'errore che ottengo è _Refused to load the image 'https://MY_URL/t/e9a21273-bdfd-4f85-9624-0dfaff99d8d2/s/image.jpg' because it violates the following Content Security Policy directive: "img-src 'self'"._

**Comment by evilaliv3** [comment, marked as answer]
@maelaborazioni : intuisco dal link che stai mettendo che tiu stia accedendo alla istanza secondaria tramite il dominio dell'istanza primaria.

Questo tipo di collegamento è utile solo per finalità di configurazione ma non ti permette di testare ne le cofigurazioni CSS ne Javascript.

Questo limite è imposto per ragioni di sicurezza in quanto permetterlo reppresentrebbe una significativa possibilità di attacco da una istanza figlio alla istanza principale.

Per poter vedere dunque le configurazioni devi accedere alla tenant secondaria tramite il suo apposito dominio

**Comment by maelaborazioni** [comment]
Chiarissimo, usando l'admin del sito figlio sono riuscito in tempo zero.
Grazie mille per il supporto!
